modules/docs/lib/widgets/model.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import {ModelHookValue, ModelValue} from '../../docgen/plugins/customTypes';
+import {PropertiesTable, registerWidget, Value} from '../Value';
+import {Heading} from '../widgetUtils';
+
+registerWidget<ModelValue>('model', ({value}) => {
+  return (
+    <>
+      <Heading>State</Heading>
+      <PropertiesTable properties={value.state} showDefault={false} />
+      <Heading>Events</Heading>
+      <PropertiesTable properties={value.events} showDefault={false} />
+      <Heading>Additional Properties</Heading>
+      <PropertiesTable properties={value.modelProperties} showDefault={false} />
+    </>
+  );
+});
+
+registerWidget<ModelHookValue>('modelHook', ({value, doc}) => {
+  return (
+    <>
```

Comment:
unnecessary fragment wrapping `<code>`

Reply:
sorry i know these fragment comments should be a lint rule, i'm not worried if you ignore them